src/managed_data/client.py

```diff
@@ -11,7 +12,18 @@
 
 
 @dataclass
-class DeepOriginClient:
+class Client(ABC):
```

Comment:
For thought: Why add this extra layer of abstraction? What other clients do you envision?

Reply:
the only other "client" is the mock client used in testing. previously it inherited from `DeepOriginClient` and overwrote those methods, which felt a little confusing.